reconcile/test/test_utils_ldap_client.py

```diff
@@ -0,0 +1,23 @@
+from ldap3 import Connection
+from reconcile.utils.ldap_client import LdapClient
+
+
+def test_ldap_client(mocker):
```

Comment:
Maybe this is `test_ldap_client_from_settings` and we also want to test that it works as expected when initialized directly?

Reply:
I added a test for direct initialization but I'm not really sure the value it adds.